.github/workflows/govulncheck.yaml

```diff
@@ -0,0 +1,122 @@
+on:
+  workflow_call:
+    inputs:
+      instance:
+        required: true
+        type: string
+
+name: Govulncheck check
+jobs:
+  govulncheck:
+    runs-on: ${{ inputs.instance }}
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install golang
+        run: |
+          ./tests/install_go.sh -f -p
+          echo "/usr/local/go/bin" >> "$GITHUB_PATH"
+
+      - name: Install govulncheck
+        run: |
+          go install golang.org/x/vuln/cmd/govulncheck@latest
+          echo "${HOME}/go/bin" >> "$GITHUB_PATH"
+
+      - name: Build all runtime binaries
+        run: |
+          cd src/runtime
+          make all
+        env:
+          SKIP_GO_VERSION_CHECK: "1"
```

Comment:
I think it's better to use the output tarball from `publish-kata-deploy-payload-amd64` here. This way we know everything was built according to how the CI expects to build.

Reply:
I see the point of using prebuilt binaries here. But I think it might be better to use a Makefile to build the binaries on the go for two reasons. 1. this keeps the door open for adding govulncheck source mode in the future if needed, as govulncheck is still under development. 2. this approach follows the existing static analysis workflow pattern of building with a Makefile, similar to how rust clippy checks are run from make check.